JapaneseFilter/sections/specific.txt

```diff
@@ -44,6 +44,8 @@ oppai-paipai.com##.movie-player-block
 ||js.gazo.space/tagjpa.php?
 cndata.jpg4.pw###myinvidad
 cndata.jpg4.pw###thbss > div > a[onclick][title="CLOSE"]
+fril.jp###mkdiv_header_pitari
+fril.jp##.campaignBanners
```

Comment:
Not an ad, just a PR of fundraising for Ukrine by the same entity. I don't know if worth block even as annoyance @Alex-302 @zloyden

Reply:
Current contant may be changed.

I see ads here:

<details>
<summary>Details</summary>

![image](https://user-images.githubusercontent.com/8361299/166055263-c9613c39-dd24-4a9a-b7d7-805097021ce7.png)


</details>

Reply:
They're still self-promo.